verification/src/solidity/verifier.rs

```diff
@@ -97,3 +327,329 @@ impl Verifier {
+
+    const DEFAULT_CONTRACT_NAME: &'static str = "Contract";
+
+    const DEFAULT_CONSTRUCTOR_ARGS: &'static str =
+        "0000000000000000000000000000000000000000000000000000000000000fff";
+    // {"ipfs": h'1220EB23CE2C13EA8739368F952F6C6A4B1F0623D147D2A19B6D4D26A61AB03FCD3E', "solc": 0.8.14}
+    const DEFAULT_ENCODED_METADATA_HASH: &'static str = "a2646970667358221220eb23ce2c13ea8739368f952f6c6a4b1f0623d147d2a19b6d4d26a61ab03fcd3e64736f6c634300080e0033";
+    const DEFAULT_BYTECODE_WITHOUT_METADATA_HASH: &'static str = "608060405234801561001057600080fd5b5060405161022038038061022083398101604081905261002f91610074565b600080546001600160a01b0319163390811782556040519091907f342827c97908e5e2f71151c08502a66d44b6f758e3ac2f1de95f02eb95f0a735908290a35061008d565b60006020828403121561008657600080fd5b5051919050565b6101848061009c6000396000f3fe608060405234801561001057600080fd5b50600436106100365760003560e01c8063893d20e81461003b578063a6f9dae11461005a575b600080fd5b600054604080516001600160a01b039092168252519081900360200190f35b61006d61006836600461011e565b61006f565b005b6000546001600160a01b031633146100c35760405162461bcd60e51b815260206004820152601360248201527221b0b63632b91034b9903737ba1037bbb732b960691b604482015260640160405180910390fd5b600080546040516001600160a01b03808516939216917f342827c97908e5e2f71151c08502a66d44b6f758e3ac2f1de95f02eb95f0a73591a3600080546001600160a01b0319166001600160a01b0392909216919091179055565b60006020828403121561013057600080fd5b81356001600160a01b038116811461014757600080fd5b939250505056fe";
+    const DEFAULT_DEPLOYED_BYTECODE_WITHOUT_METADATA_HASH: &'static str =  "608060405234801561001057600080fd5b50600436106100365760003560e01c8063893d20e81461003b578063a6f9dae11461005a575b600080fd5b600054604080516001600160a01b039092168252519081900360200190f35b61006d61006836600461011e565b61006f565b005b6000546001600160a01b031633146100c35760405162461bcd60e51b815260206004820152601360248201527221b0b63632b91034b9903737ba1037bbb732b960691b604482015260640160405180910390fd5b600080546040516001600160a01b03808516939216917f342827c97908e5e2f71151c08502a66d44b6f758e3ac2f1de95f02eb95f0a73591a3600080546001600160a01b0319166001600160a01b0392909216919091179055565b60006020828403121561013057600080fd5b81356001600160a01b038116811461014757600080fd5b939250505056fe";
+
+    const DEFAULT_CREATION_TX_INPUT: &'static str = concatcp!(
+        DEFAULT_BYTECODE_WITHOUT_METADATA_HASH,
+        DEFAULT_ENCODED_METADATA_HASH,
+        DEFAULT_CONSTRUCTOR_ARGS
+    );
+    const DEFAULT_DEPLOYED_BYTECODE: &'static str = concatcp!(
+        DEFAULT_DEPLOYED_BYTECODE_WITHOUT_METADATA_HASH,
+        DEFAULT_ENCODED_METADATA_HASH
+    );
+
+    #[test]
+    fn test_initialization_with_valid_data() {
+        let verifier = Verifier::new(
+            DEFAULT_CONTRACT_NAME.to_string(),
+            None,
+            DEFAULT_CREATION_TX_INPUT,
+            DEFAULT_DEPLOYED_BYTECODE,
+        );
+        assert!(verifier.is_ok(), "Initialization with \"0x\" prefix failed");
```

Comment:
why "with 0x prefix"? probably you mixed it up with next test

Reply:
Fixed, thanks!